scripts/install/install-pocs.sh

```diff
@@ -248,11 +259,15 @@ function do_install {
     echo "OS: ${OS}"
     echo "Logfile: ${LOGFILE}"
 
-    exit 0;
+
 
     echo "Creating directories in ${PANDIR}"
     make_directories
 
+    echo "Setting up environment variables"
```

Comment:
Let's tell them where they end up.

```suggestion
    echo "Setting up environment variables in ${ENV_FILE}"
```